fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -29,17 +33,38 @@ class ConditionalSelectionRate(ClassificationMoment):
 
     """
 
+    def __init__(self, ratio=1.0):
```

Comment:
On one hand, I think it would be nice to just do `DemographicParity`, which would default to difference unless 'ratio' is specified.

On the other hand, we have examples such as `AccuracyScoreDifference` and `AccuracyScoreRatio` which would be confusing to handle by a class called `AccuracyScore` (and similarly for `ErrorRate`), because that strongly evokes the overall performance metric... so I think I'm in favor of `<Metric>Ratio` and `<Metric>Difference` approach.

Reply:
that said... let's hold off on the renaming for a brief moment and iron everything else first.

Reply:
That's what I thought. We can deprecate the existing "DemographicParity", etc. in favor of <metric>Difference, <metric>Ratio...